spec/javascripts/health-care/components/form-elements/ErrorableInput.spec.jsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import SkinDeep from 'skin-deep';
+
+import ErrorableInput from '../../../../../_health-care/_js/_components/_form-elements/ErrorableInput';
+
+describe('<ErrorableInput>', () => {
+  describe('propTypes', () => {
+    let consoleStub;
+    beforeEach(() => {
+      consoleStub = sinon.stub(console, 'error');
+    });
+
+    afterEach(() => {
+      consoleStub.restore();
+    });
+
+    it('label is required', () => {
```

Comment:
This whole set of tests is a little OCD. propType generates warnings in console in _dev_ mode for React when the Component API constraints might be violated.

What this is testing is effectively that someone said `label: React.PropTypes.[something].isRequred`